googleapis/google-cloud-node
# migrate code from googleapis/nodejs-service-control

- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- chore: add initial files for launch
- feat!: initial stub of library
- feat: add initial samples and tests (#2)
- chore: release 0.2.0 (#13)
- build: update package-name, update links to service control and service management (#17)
- chore: migrate to owl-bot (#18)
- chore(deps): update dependency @types/sinon to v10 (#21)
- chore(deps): update dependency ts-loader to v9 (#25)
- fix(deps): require google-gax v2.12.0 (#30)
- fix: use require() to load JSON protos (#32)
- chore: new owl bot post processor docker image (#34)
- chore: update gapic-generator-typescript to v1.3.2 (#33)
- chore: release 1.0.0 (#38)
- chore(deps): update dependency sinon to v11 (#39)
- fix: add dependency log_severity.proto (#48)
- feat: Added the gRPC service config for the Service Controller v1 API docs: Updated some comments. (#50)
- chore: release 1.1.0 (#52)
- fix(deps): require google-gax v2.17.0 (#54)
- chore: release 1.1.1 (#56)
- fix(deps): google-gax v2.17.1 (#57)
- chore: release 1.1.2 (#58)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#59)
- chore: release 1.1.3 (#60)
- chore(nodejs): update client ref docs link in metadata (#65)
- fix(deps): google-gax v2.24.1 (#67)
- chore: release 1.1.4 (#68)
- feat: turns on self-signed JWT feature flag (#69)
- chore: release 1.2.0 (#70)
- chore: disable renovate dependency dashboard (#1194) (#72)
- chore: fix broken endpoints test (#74)
- fix(build): set default branch to main (#75)
- chore: release 1.2.1 (#76)
- build(node): update deps used during postprocessing (#1243) (#83)
- docs: fix docstring formatting and add generated samples (#84)
- chore(cloud-rad): delete api-extractor config (#85)
- chore: fix the wrong post processor image (#87)
- chore(deps): update dependency @types/node to v16 (#86)
- chore(deps): update dependency sinon to v12 (#88)
- docs(samples): add example tags to generated samples (#89)
- build: add generated samples to .eslintignore (#90)
- docs(node): support "stable"/"preview" release level (#1312) (#93)
- chore: add api_shortname and library_type to repo metadata (#92)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#95)
- test(nodejs): remove 15 add 16 (#1322) (#97)
- build: update copyright year to 2022 (#100)
- chore: update v2.12.0 gapic-generator-typescript (#102)
- docs(nodejs): version support policy edits (#1346) (#104)
- chore: proto cleanup (#105)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#106)
- feat: add KMS configuration in settings (#107)
- chore(main): release 1.3.0 (#108)
- chore(deps): update dependency sinon to v13 (#111)
- docs: fix relative and broken links (#120)
- feat: promote to stable (#122)
- chore(main): release 1.4.0 (#123)
- build(node): update client library version in samples metadata (#1356) (#130)
- chore(deps): update dependency @types/mocha to v9 (#131)
- chore(deps): update dependency sinon to v14 (#134)
- build!: update library to use Node 12 (#135)
- chore(main): release 2.0.0 (#137)
- chore(deps): update dependency jsdoc-region-tag to v2 (#141)
- chore(deps): update dependency jsdoc-fresh to v2 (#140)
- fix(docs): describe fallback rest option (#143)
- chore(main): release 2.0.1 (#144)
- chore(deps): update dependency linkinator to v4 (#148)
- chore: remove unused proto imports (#149)
- fix: better support for fallback mode (#150)
- fix: change import long to require (#151)
- fix: do not import the whole google-gax from proto JS (#1553) (#154)
- fix: allow passing gax instance to client constructor (#155)
- chore(main): release 2.0.2 (#152)
- fix: preserve default values in x-goog-request-params header (#156)
- test: use fully qualified request type name in tests (#159)
- fix(deps): use google-gax v3.5.2 (#163)
- chore(deps): update dependency @types/node to v18 (#161)
- chore(deps): update dependency jsdoc to v4 (#165)
- fix: regenerated protos JS and TS definitions (#166)
- chore(main): release 2.0.3 (#157)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
